content/middlewares.md

```diff
@@ -177,6 +177,33 @@ export class AppModule {
 
 > info **Hint** The `apply()` method may either take a single middleware, or multiple arguments to specify <a href="/middleware#multiple-middleware">multiple middlewares</a>.
 
+#### Middleware versioning
```

Comment:
Instead of adding this section to the Middleware doc, we should consider adding it here https://docs.nestjs.com/techniques/versioning (to be consistent with ctrl versioning etc)